mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/StatefulContestScreenTest.kt

```diff
@@ -91,4 +96,149 @@ class StatefulContestScreenTest {
       rule.onNodeWithText(strings.tournamentsBadgeParticipant).assertIsDisplayed()
     }
   }
+
+  @Test
+  fun given_statefulContestScreen_when_tournamentPresent_then_correctTimeDisplayed() {
+    runTest {
+      val auth = buildAuth { user("email@example.org", "password", "1") }
+      val dataStoreFactory = emptyDataStoreFactory()
+      val store = buildStore {
+        collection("users") { document("1", ProfileDocument("1", name = "A")) }
+        collection(TournamentDocument.Collection) {
+          document(
+              "id1",
+              TournamentDocument(
+                  "tid1",
+                  "2",
+                  "Tournament 1",
+                  playerIds = listOf("1"),
+                  creationTimeEpochMillis = 0L))
+        }
+      }
+
+      val time = FakeTimeProvider
+      val assets = emptyAssets()
+      val authFacade = AuthenticationFacade(auth, store)
+      val socialFacade = SocialFacade(auth, store)
+      val chessFacade = ChessFacade(auth, store, assets)
+      val speechFacade = SpeechFacade(FailingSpeechRecognizerFactory)
+      val tournamentFacade = TournamentFacade(auth, dataStoreFactory, store, time)
+
+      val user = authFacade.currentUser.filterIsInstance<AuthenticatedUser>().first()
```

Comment:
You never logged in, but actually can still fetch the `currentUser`. This looks like a bug in `FakeStore` which we can fix later.